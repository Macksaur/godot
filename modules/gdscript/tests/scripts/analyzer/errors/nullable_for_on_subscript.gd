class A:
    var arr: Array? = null
func test():
    var instance := A.new()
    for element in instance.arr: # Should error
        print(element)