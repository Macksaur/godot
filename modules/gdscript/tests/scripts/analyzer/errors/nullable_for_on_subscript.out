GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable expression "instance.arr" to be used in a for-loop.
